### PR TITLE
Feature/update lobby

### PR DESCRIPTION
**Pull request by vetom**
Now the lobby is showing the players name.

### CODE REVIEW

File: src/views/Lobby/index.js [hunk 1]
```diff
@@ -28,8 +28,9 @@ class Lobby extends Component {
     const { navigation } = this.props;
     const { party = {} } = navigation.state.params;
     if (party) {
-      const { id, players } = party;
-      this.setState({ id, players: Object.keys(players) });
+      const { id, players = {} } = party;
+      const playersNames = Object.keys(players).map(p => players[p].name);
+      this.setState({ id, players: playersNames });
       this.unsubscribe = this.db.subscribeToParty(id, p => this.updateState(p));
```
Comment (wovalle) [inline src/views/Lobby/index.js]: Must be refactored to use global state

File: src/actions/party.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const noop = () => {};
+
+let unsubscribeParty = noop;
+
+export const setParty = party => ({
+  type: types.party.set,
+  party,
+});
+
+export const unsubscribe = () => {
```
Comment (mamodom) [inline src/actions/party.js]: You should assign `noop` to `unsubscribeParty` after unsubscribing.

File: src/reducers/party.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import types from '../actions';
+
+const party = (state = {}, action) => {
+  switch (action.type) {
+    case types.party.set:
+      return Object.assign({}, state, action.party);
```
Comment (mamodom) [inline src/reducers/party.js]: As `setParty` is meant to replace the current party you could just return `action.party`

File: src/actions/party.js (hunk 2 above)
Comment (vetom) [reply src/actions/party.js]: Good one, it's done in line 18.

**Pull request merged**